config/prod/tom-cvat-ubuntu.yaml

```diff
@@ -0,0 +1,53 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-ubuntu-v2.j2
+stack_name: tom-cvat-ubuntu
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "CompOnc"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "dream"
+  # The resource owner
+  OwnerEmail: "thomas.yu@sagebase.org"
+
+  # Default account settings, leave alone unless you plan to override
+  VpcName: "{{stack_group_config.default_vpc}}"
+  KeyName: "{{stack_group_config.default_key_pair}}"
+  AMIId: "{{stack_group_config.default_ubuntu_ami}}"
+
+  # Settings have default values but can be overriden. Set the below
+  # parameters *only* if you want to override the defaults.
+
+  # (Optional) EC2 instance type, default is "t3.nano" (other available types https://aws.amazon.com/ec2/pricing/on-demand/)
+  InstanceType: "p3.2xlarge"
+  # (Optional) EC2 boot volume size in GB, default is 8GB, Max is 1000GB
+  VolumeSize: "500"
+  # (Optional) Base instance on AMI (default: ami-0a313d6098716f372)
+  # AMIId: "ami-0565af6e282977273"
+  # (Optional) Run EC2 in a subnet (default: PrivateSubnet) (valid values: PrivateSubnet or PublicSubnet)
+  # VpcSubnet: "PublicSubnet"
+  # (Optional) Set true to enable daily backups (default: false)
+  # BackupEc2: "true"
+  # (Optional) Number of daily backups to keep (default: 7) (valid range: 1-180)
+  # SnapshotCount: "14"
+  Ec2InstanceId: "i-03510ca53eb38da83"
+  SSLCertificateIdArn: "arn:aws:acm:us-east-1:055273631518:certificate/992c2d49-579a-465e-beff-1a08c284f7db"
+  VpcId: "vpc-e66bd19d"   # computevpc
+  AppPort: "80"         # rstudio port
+  Subnets: "subnet-21a68d7c,subnet-4f3e2b2b,subnet-77d1e458"
+  # Integration with our jumpcloud directory service (do not change)
+  JcConnectKey: !ssm /infra/JcConnectKey
+  JcServiceApiKey: !ssm /infra/JcServiceApiKey
+  JcSystemsGroupId: !ssm /infra/JcSystemsGroupId
+
+sceptre_user_data:
+  # (Optional) Expose ports to incoming traffic (default is no open ports) (valid range: 1-65535)
+  OpenPorts: ["80","8082"]
```

Comment:
Will my shiny application still be able to communicate with my CVAT application which is hosted at 8082

Reply:
shiny and cvat are both docker apps running on the same instance so you can manage the ports between them yourself.